commit e1cafe5d2123631cdff8c115de15575bc6643677
Author: Loren Segal <lsegal@soen.ca>
Date:   Sun Jan 23 04:05:41 2011 -0500

    Fix failing spec when ~/.yard/ignored_plugins does not exist

diff --git a/spec/config_spec.rb b/spec/config_spec.rb
index 8aea427..5bcaa60 100644
--- a/spec/config_spec.rb
+++ b/spec/config_spec.rb
@@ -17,7 +17,7 @@ describe YARD::Config do
     
     it "should overwrite options with data in ~/.yard/config" do
       File.should_receive(:file?).with(YARD::Config::CONFIG_FILE).and_return(true)
-      File.should_receive(:file?).with(YARD::Config::IGNORED_PLUGINS).and_return(true)
+      File.should_receive(:file?).with(YARD::Config::IGNORED_PLUGINS).and_return(false)
       YAML.should_receive(:load_file).with(YARD::Config::CONFIG_FILE).and_return({'test' => true})
       YARD::Config.load
       YARD::Config.options[:test].should be_true
